pkgs/development/beam-modules/build-mix.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenv, writeText, elixir, erlang, hex, lib }:
+
+{ name
+, version
+, src
+, setupHook ? null
+, buildInputs ? []
+, beamDeps ? []
+, postPatch ? ""
+, compilePorts ? false
+, installPhase ? null
```

Comment:
maybe we just remove this arguments and just
```
configurePhase = attrs.configurePhase or ''...'';
```
see mix-release.
Let me know if you want me to clarify further